### DISCUSSION TITLE
134

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#752] [Comments]
http://jojoldu.tistory.com/134

**Comment by jojoldu** [comment]
작성자: Havi

좋은 글 잘보았습니다. 처음 보는 어노테이션들이 난무하는...ㅋㅋ
직접 적용해서 공부좀 해봐야 겠네요ㅎㅎ

**Comment by jojoldu** [comment]
작성자: _Jbee

좋은 글 감사합니다~

**Comment by KKambi** [comment]
SalesConverter는 상태(값)를 갖고 있지 않으며, input이 같을때 output값도 항상 같기에 static 메소드로 구성하였습니다.

이 부분에서 왜 인풋이 같을 때 항상 동일한 아웃풋이 보장되면 static method를 고려하는건가요??

**Comment by kimyuil** [comment]
공부하다 우연히 들어왔는데 향로님이시네요 ㅎㅎ 유튜브 잘보고있습니다 반갑습니다~

**Comment by movejae** [comment]
[문제 상황] 첫번째 그림에 PAYMENT테이블의 (ID) 5 의 (OWNER_ID) 가  1로 되있는데 잘못된건가요?

실제 ApplicationTest.java 의 setup()메소드에서는  2(L) 로 되어있네용

**Comment by ImRieul** [comment]
Open API를 가져오면서 key값을 수정하고 클래스로 저장하고 싶어 알아보던 도중 보게 되었습니다. 좋은 글 감사합니다..!!

**Comment by zhtmr** [comment]
wow
